README.md

```diff
@@ -16,7 +16,7 @@ Kotlin wrappers are generated with [uniffi-kotlin-multiplatform-bindings](https:
 The Swift wrappers are distributed as a Swift Package.
 To use the wrappers in your project, add the following dependency to your `Package.swift`:
 ```swift
-    .package(url: "https://github.com/hyperledger/aries-uniffi-wrappers", from: "0.1.0"),
+    .package(url: "https://github.com/LF-Decentralized-Trust-labs/aries-uniffi-wrappers", from: "0.1.0"),
```

Comment:
I've tested the repo change in Swift, and it worked well without this change thanks to the redirection.